Graphs/DynamicBarGraph/index.js

```diff
@@ -139,7 +140,12 @@ class BarGraph extends XYGraph {
       stack: this.stack,
       otherOptions,
       stackSequence,
-    })
+    });
+  
+    if (xTicksLabel) {
```

Comment:
If the user misses to include a specific xTicksLabel, that bar itself will not get rendered.
For example with, 
```
"xTicksLabel": {
            "PG1": "ONE",
            "PG2": "TWO",
            "PG3": "THREE",
            "PG4": "FOUR"
        }
```
we have,
<img width="465" alt="Screen Shot 2019-09-05 at 7 45 31 AM" src="https://user-images.githubusercontent.com/24920919/64352607-2e08cb00-cfb1-11e9-8441-2a41652a0000.png">

But if user misses to provide mapping for PG2, we now have the same graph as,
<img width="466" alt="Screen Shot 2019-09-05 at 7 46 51 AM" src="https://user-images.githubusercontent.com/24920919/64352704-5a244c00-cfb1-11e9-88bf-0f9514db6b8a.png">

In my opinion, the bar should still be rendered, but with xTicksLabel as constructed from response data